### ISSUE TITLE
Decouple the AdobePdf module from ScholarSphere

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/scholarsphere#1638]
It might be useful to have this as its own gem so it can be used elsewhere.  It should be pretty close to decoupling.  Any place that invokes `AccessibilityCheckResult.create!` can be replaced with either the returned report or an error.  File handling will likely need changed.  The `S3Handler` module requires a `FileResource`, which binds it to ScholarSphere.  Perhaps just handing it an S3 key, or file path would be better for decoupling?
